network/setup.yml

```diff
@@ -67,3 +69,27 @@ controller_templates:
             - prefix_lists
             - snmp
             - user
+
+  - name: "NETWORK / Report"
+    job_type: check
```

Comment:
Does check mode matter? check and run will both gather facts without making changes right?

Reply:
I believe so, not 100% sure though.